core/src/main/scala/kafka/security/auth/SimpleAclAuthorizer.scala

```diff
@@ -198,31 +202,54 @@ class SimpleAclAuthorizer extends Authorizer with Logging {
     }
   }
 
+  def getMatchingAcls(resourceType: ResourceType, resourceName: String): Set[Acl] = {
+    val filter = new ResourceFilter(resourceType.toJava, resourceName, JResourceNameType.ANY)
+
+    inReadLock(lock) {
+      aclCache
+        .filterKeys(aclResource => AclUtils.matchResource(aclResource.toJava, filter))
+        .flatMap(_._2.acls)
```

Comment:
@junrao, if you can suggest an alternative to this `flatmap(_._2.acls)` I'm happy to plug it in, but my Scala's a little rusty.